### PR TITLE
Adds xml response

### PR DESCRIPTION
**Pull request by VictorHarbo**
Adds backwards compatibility with xml response support

### CODE REVIEW

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 1]
```diff
@@ -442,6 +442,14 @@ paths:
...
+      
+      - name: format
+        in: path
+        description: "The service returns technical information about the requested image in JSON format. XML is also supported as it was used in version one of the IIIF API"
+        required: true
+        schema:
+          type: string
+          enum: ["json", "xml"]
 
       x-streamingOutput: true
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:454]: Streaming output is for "large" results: Think binaries or hundreds of records. It should be removed here

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 2]
```diff
@@ -442,6 +442,14 @@ paths:
         required: true
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:442]: Re: `description`: Remember to mention that the OpenAPI-GUI takes care of encoding when experimenting.

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 3]
```diff
@@ -451,7 +459,10 @@ paths:
             content: 
               application/ld+json:
                 schema:
-                  $ref: '#/components/schemas/jsonld'
+                  $ref: '#/components/schemas/IIIF_info'
+              application/xml:
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:463]: `xml` har brug for et ydre element of i dette tilfælde for et namespace. Det kan angives under `IIIF_info` component med

```
      xml:
        name: info
        namespace: 'http://library.stanford.edu/iiif/image-api/ns/'
```

**Review by tokee — CHANGES_REQUESTED** [review]
Minor corrections to OpenAPI mechanisms

File: src/main/openapi/ds-image-openapi_v1.yaml (hunk 2 above)
Comment (VictorHarbo) [reply src/main/openapi/ds-image-openapi_v1.yaml:442]: Description has been updated to include information on GUI handling encoding

File: src/main/openapi/ds-image-openapi_v1.yaml (hunk 3 above)
Comment (tokee) [reply src/main/openapi/ds-image-openapi_v1.yaml:462]: Unfortunately it does not gets assigned as a default namespace. Bu we'll postpone this bug 'till later.

**Pull request merged**